sdk/resourcemanager/azure-resourcemanager-resources/pom.xml

```diff
@@ -44,6 +44,8 @@
       --add-opens com.azure.resourcemanager.resources/com.azure.resourcemanager.resources=ALL-UNNAMED
       --add-opens com.azure.core/com.azure.core.http=com.azure.resourcemanager.resources
       --add-opens com.azure.core/com.azure.core.http.policy=com.azure.resourcemanager.resources
+
+      --add-opens com.azure.core/com.azure.core.implementation.util=ALL-UNNAMED
```

Comment:
Do we need to wait for Liudmila's response for this?

Reply:
asked her in chat

will wait for tomorrow to merge. anyway this is only a test case.

Reply:
Liudmila confirmed that there is no change to core or http-client-netty. Hence we still need it in LIVE or RECORD.